tests/api.c

```diff
@@ -10369,6 +10514,28 @@ static void test_wolfSSL_set_options(void)
     #endif /* defined(OPENSSL_EXTRA) && !defined(NO_CERTS) && \
              !defined(NO_FILESYSTEM) && !defined(NO_RSA) */
 }
+	  #if defined(OPENSSL_EXTRA) && !defined(NO_CERTS)
+static int verify_cb(int ok, X509_STORE_CTX *ctx)
+{
+	  (void) ok;
+		(void) ctx;
```

Comment:
Can you adjust spacing here?  These few lines look to be inconsistent.

Reply:
FIxed